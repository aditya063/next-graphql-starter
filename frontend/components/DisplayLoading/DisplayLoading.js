import * as sc from './DisplayLoading.style';

const DisplayLoading = () => <sc.DisplayLoading>Loading...</sc.DisplayLoading>;

export default DisplayLoading;
