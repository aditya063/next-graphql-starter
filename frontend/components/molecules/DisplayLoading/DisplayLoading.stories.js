import DisplayLoading from './DisplayLoading';

export default { title: 'Molecules', component: DisplayLoading };

export const displayLoading = () => <DisplayLoading />;
