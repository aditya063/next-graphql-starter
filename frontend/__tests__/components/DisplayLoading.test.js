import React from 'react';
import { shallow } from 'enzyme';
import DisplayLoading from '../../components/DisplayLoading/DisplayLoading';

describe('DisplayLoading', () => {
  let mockProps;
  let wrapper;

  beforeEach(() => {
    jest.resetAllMocks();
    mockProps = {};
    wrapper = shallow(<DisplayLoading {...mockProps} />, {
      disableLifecycleMethods: true
    });
  });

  it('matches snapshot', () => {
    expect(wrapper.text()).toContain('Loading ...');
    expect(wrapper).toMatchSnapshot();
  });
});
