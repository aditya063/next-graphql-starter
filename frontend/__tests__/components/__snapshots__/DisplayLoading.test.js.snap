// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`DisplayLoading renders correctly 1`] = `
<p>
  Loading...
</p>
`;
